pages/models.py

```
@@ -0,0 +1,11 @@
+from django.db import models
+
+# Create your models here.
+
+
+def add_note(note, notes=[]):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)